website/docs/guides/visual-editor-qs.md

```diff
@@ -0,0 +1,287 @@
+
+- Accessing the Visual Editor and creating a new model
+- How to navigate the interface
+- How to build a model using operators
+- How to commit your changes to Git
+- Locating your Visual Editor model and data
+
+## Prerequisites
+
+To use the Visual Editor, you must meet the following prerequisites:
+
+- Be on an [Enterprise dbt Cloud account](/faqs/Accounts/cloud-upgrade-instructions#enterprise-plan).
+- Your account must have the following configured:
+    - A data warehouse connection
+    - Integration with a Git provider
+- Source models for the Visual Editor must have been run at least once
+- You must have a `developer` license
+- You must have credentials configured for your data warehouse and Git provider in the **Your profile** section of the **Account settings**. 
+
+:::note
+
+The examples in this guide use the [Jaffle Shop](https://github.com/dbt-labs/jaffle-shop) GitHub repo sample project. You can use your own data, but the Jaffle Shop offers a full-featured project useful for testing dbt features. Ask your dbt Cloud administrator about importing it to a project in your environment. They must also execute `dbt run` on the Jaffle Shop project in your `Production` environment before you begin, or you will be unable to reference the source models.
+
+:::
+
+## Access the Visual Editor
+
+To access the Visual Editor:
+
+1. Click **Develop** from the main menu. If you do not see the **Develop** option, ensure you have selected a **Project** from the menu.
+2. Click **Visual Editor**.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-main-menu.png" width="90%" title="Visual Editor in the main menu."/>
+
+3. From the right side, click **Get started** and then click **Create new model**.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-create-new-model.png" width="90%" title="Create a new model from the Visual Editor landing page."/>
+
+## Navigating the interface
+
+The Visual Editor comprises a series of menus activated by clicking icons surrounding the border of the larger canvas. With none of the menu items activated, the workspace looks like this:
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-screen.png" width="90%" title="The Visual Editor canvas. The number of items is defined in this section." />
+
+Click on an icon to expand its section or execute an action depending on its purpose. The options are as follows:
+
+1. The model's title. This defaults to "Untitled" but can be edited anytime by clicking on it.
+2. The **Operators** toolbar contains the building blocks for creating a model with the editor. 
+3. The **SQL code** area displays the SQL that compiles your model.
+4. The **Runs and previews** that displays run data and previews data for individual operators.
+5. The **Commit history** display.
+6. The navigation tab that has icons for (from top to bottom):
+    - Zoom in
```

Comment:
I see 5 icons in the screen shot and only 4 descriptions.

Reply:
Good catch! They added a new option